app/matplotlib.ipynb

```diff
@@ -47,21 +43,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
-    "collapsed": true,
-    "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
-    }
+    "collapsed": true
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "plt.figure(figsize=(12, 6))\n",
+    "\n",
+    "plt.title(\"Population\")\n",
+    "plt.plot(years, population)"
```

Comment:
The x and y labels are missing for the line plot. Add `plt.xlabel("Year")` and `plt.ylabel("Population")` to meet the task requirements.